gpg-enter-passphrase-reenter-20190506.json

```diff
@@ -0,0 +1,17 @@
+{
+  "area": [
+    {
+      "xpos": 301,
+      "ypos": 334,
+      "width": 253,
+      "height": 18,
+      "type": "match",
+      "match": 75,
+      "margin": 150
+    }
+  ],
+  "properties": [],
+  "tags": [
+    "gpg-enter-passphrase-reenter"

```

Comment:
I get your point you want to follow same pattern , but for me it sounds too weird . maybe we fallback just to `gpg-reenter-passphrase` ?

Reply:
or you can use as main pattern `gpg-passphrase` : 

- `gpg-passphrase-enter`
- `gpg-passphrase-reenter`
- `gpg-passphraseunlock`

Reply:
I followed your second suggestion, I have also change the test case.